SQL assignment.txt

```diff
@@ -0,0 +1,1533 @@
+1) List all the columns of the Salespeople table. 
+
+mysql> DESC SALESPEOPLE;
```

Comment:
This is not the query for listing columns.

Reply:
resolved